storage/rocksdb/ha_rocksdb.cc

```diff
@@ -7549,6 +7576,7 @@ int ha_rocksdb::create_table(const std::string &table_name,
 
   /* Create table/key descriptions and put them into the data dictionary */
   m_tbl_def = new Rdb_tbl_def(table_name);
+  //m_tbl_def->set_create_time(time(nullptr));
```

Comment:
Most of these changes can be removed?

Reply:
Yes, now done.  (they were actually gone in MariaDB version. I'm sorry for mixing something up and still having them there)